### PR TITLE
Polish the upload overlay

### PR DESCRIPTION
**Pull request by JammingBen**
## Description

* Polished the design of the overlay a bit
* Header now shows the amount of successful uploads

Followup for https://github.com/owncloud/web/pull/6202

## Related Issue
- Part of https://github.com/owncloud/web/issues/6819

## Screenshots:
image

### CODE REVIEW

**Comment by JammingBen** [comment]
I'm not super happy about the handling of failed uploads for several reasons:

* Uploading 3 files where 1 upload fails -> it says "0 of 1 file uploaded", although it should be "2 of 3 files uploaded" or even better "1 of 3 files failed to upload"
* It would be nice to know which files failed to upload
* Design is "okay-ish"

Unfortunately, this part comes from the Uppy StatusBar plugin, so we can't easily change that. For the future we might kick that plugin and implement our own solution. But for now we're fine IMO.

File: packages/web-runtime/src/components/UploadInfo.vue [hunk 1]
```diff
@@ -191,33 +191,70 @@ export default {
...
+    .uppy-StatusBar.is-error,
+    .uppy-StatusBar.is-uploading {
+      margin-top: var(--oc-space-medium);
+    }
+    .uppy-StatusBar.is-error .uppy-StatusBar-progress {
+      background-color: var(--oc-color-swatch-danger-default);
+    }
+    .uppy-StatusBar.is-error .uppy-StatusBar-statusIndicator {
+      color: var(--oc-color-swatch-danger-default);
+      width: 8px;
```
Comment (kulmann) [inline packages/web-runtime/src/components/UploadInfo.vue:251]: you could use `var(--oc-space-small)` instead of `8px`

**Review by kulmann — APPROVED** [review]
cool 👍

**Comment by ownclouders** [comment]
Results for <strong>e2e-tests oCIS</strong> https://drone.owncloud.com/owncloud/web/25035/12/1
<details><summary>:boom: To see the trace, please open the link in the console ...</summary>

<p>


#### npx playwright show-trace https://cache.owncloud.com/public/owncloud/web/25035/tracing/unstructured-collection-of-testable-space-interactions-alice-2022-4-29-09-47-51.zip 


</p></details>

**Pull request merged**